backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##church-info.church-info.json

```diff
@@ -194,7 +194,7 @@
         [
           {
             "name": "PrivatePolicy",
```

Comment:
Can you make the size 100%? Also, disable localization for the Private Policy?

Reply:
Make the size 100%

Reply:
Disabled localization for Private Policy

Reply:
Great work :)